app/src/main/java/org/oppia/app/player/state/StateFragmentPresenter.kt

```diff
@@ -81,7 +81,8 @@ class StateFragmentPresenter @Inject constructor(
 
   private var showCellularDataDialog = true
   private var useCellularData = false
-  private var explorationId: String? = null
+  private lateinit var explorationId: String
+  private val entityType: String = "exploration"
```

Comment:
It actually might be nice to inject this, instead. Then we could have a qualifier bind the value to the Dagger graph to keep its definition in one location, e.g.:

Qualifier:
```kotlin
@Qualifier annotation class ExplorationHtmlParserEntityType
```

Module:
```kotlin
@Module
class HtmlParserEntityTypeModule {
  @Provides
  @ExplorationHtmlParserEntityType
  fun provideExplorationHtmlParserEntityType(): String {
    return "exploration"
  }
}
```

Everywhere where we currently define this, instead inject:
```kotlin
class ExampleClass @Inject(@ExplorationHtmlParserEntityType private val entityType: String) {
  ...
}
```

In general I think we should aim to keep sentinel values like this in the Dagger graph to make it easier to share them everywhere and minimize the chance of accidentally getting it wrong in one spot (or needing to update all spots if the value needs to change in the future).

Reply:
Done.